ocaml/xapi/xapi_host.ml

```diff
@@ -27,6 +27,20 @@ module D = Debug.Make (struct let name = "xapi_host" end)
 
 open D
 
+(* [take n xs] returns the first [n] elements of [xs] and the remaining
+   tail. *)
+let take n xs =
```

Comment:
`Xapi_stdext_std.Listext.List.take` is available as well.

It's mind-boggling how this is not on the standard library yet, on previous attempts, inclusion was struck down stating it was "too niche"